PatchesApi.Tests/DynamoDbMockWebApplicationFactory.cs

```diff
@@ -53,12 +54,29 @@ private static void EnsureTablesExist(IAmazonDynamoDB dynamoDb, List<TableDef> t
             {
                 try
                 {
-                    var request = new CreateTableRequest(table.Name,
-                        new List<KeySchemaElement> { new KeySchemaElement(table.KeyName, KeyType.HASH) },
-                        new List<AttributeDefinition> { new AttributeDefinition(table.KeyName, table.KeyType) },
-                        new ProvisionedThroughput(3, 3));
-                    _ = dynamoDb.CreateTableAsync(request).GetAwaiter().GetResult();
+
+                    var keySchema = new List<KeySchemaElement> { new KeySchemaElement(table.KeyName, KeyType.HASH) };
+                    var attributes = new List<AttributeDefinition> { new AttributeDefinition(table.KeyName, table.KeyType) };
+                    //if (!string.IsNullOrEmpty(table.RangeKeyName))
+                    //{
+                    //    keySchema.Add(new KeySchemaElement(table.RangeKeyName, KeyType.RANGE));
+                    //    attributes.Add(new AttributeDefinition(table.RangeKeyName, table.RangeKeyType));
+                    //}
+                    var indexKey = table.GlobalSecondaryIndexes.SelectMany(x => x.KeySchema).FirstOrDefault(y => y.KeyType == KeyType.HASH);
```

Comment:
Could change to first or default && keytype == whatever

Reply:
I don't think you can do that